src/scripting_api/app.js

```diff
@@ -479,8 +479,30 @@ class App extends PDFObject {
     /* Not implemented */
   }
 
-  execMenuItem() {
-    /* Not implemented */
+  execMenuItem(item) {
+    if (
+      [
+        "SaveAs",
+        "FirstPage",
+        "LastPage",
+        "NextPage",
+        "PrevPage",
+        "ZoomViewIn",
+        "ZoomViewOut",
+      ].includes(item)
+    ) {
+      this._send({ command: item });
+      return;
+    }
+
+    switch (item) {
+      case "FitPage":
+        this._send({ command: "zoom", value: "page-fit" });
+        break;
+      case "Print":
+        this._send({ command: "print" });
+        break;
+    }
```

Comment:
Now that the `switch` is added, can't you simply handle all of the cases within that block instead?

```javascript
    switch (item) {
      case "SaveAs":
      case "FirstPage":
      case "LastPage":
      case "NextPage":
      case "PrevPage":
      case "ZoomViewIn":
      case "ZoomViewOut":
        this._send({ command: item });
        break;

      case "FitPage":
        this._send({ command: "zoom", value: "page-fit" });
        break;
      case "Print":
        this._send({ command: "print" });
        break;
    }
```